app/presenters/templates/Faculty/default.latte

```diff
@@ -0,0 +1,13 @@
+{block content}
+
+<section class="item__list">
+    <header class="item__list__header">
+        <h1 class="item__list__title">
+            Fakulty ({count($faculties)})
+        </h1>
+
+        <a n:href="Faculty:add" class="button button--add">Přidat</a> {* Show only if user is admin. *}
```

Comment:
Při přihlášení uživatele si do identity ukládáme jenom jeho ID. Chtělo by to i jeho roli.

Reply:
Urcite... A chtelo by to uz v tehle prvni obrazovce implementovany at to mame pak vsude 😇

Reply:
Uděláme to u druhý, už mám rozpracovaný katedry. Tady je to stejně jenom jeden if.